cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -2543,180 +2543,201 @@ pStakePoolRegistrationParserRequirements envCli =
 pProtocolParametersUpdate :: Parser ProtocolParametersUpdate
 pProtocolParametersUpdate =
   ProtocolParametersUpdate
-    <$> optional pProtocolVersion
-    <*> optional pDecentralParam
-    <*> optional pExtraEntropy
-    <*> optional pMaxBlockHeaderSize
-    <*> optional pMaxBodySize
-    <*> optional pMaxTransactionSize
-    <*> optional pMinFeeConstantFactor
-    <*> optional pMinFeePerByteFactor
-    <*> optional pMinUTxOValue
-    <*> optional pKeyRegistDeposit
-    <*> optional pPoolDeposit
-    <*> optional pMinPoolCost
-    <*> optional pEpochBoundRetirement
-    <*> optional pNumberOfPools
-    <*> optional pPoolInfluence
-    <*> optional pMonetaryExpansion
-    <*> optional pTreasuryExpansion
-    <*> pure Nothing
+-- <<<<<<< HEAD
+--     <$> optional pProtocolVersion
+--     <*> optional pDecentralParam
+--     <*> optional pExtraEntropy
+--     <*> optional pMaxBlockHeaderSize
+--     <*> optional pMaxBodySize
+--     <*> optional pMaxTransactionSize
+--     <*> optional pMinFeeConstantFactor
+--     <*> optional pMinFeePerByteFactor
+--     <*> optional pMinUTxOValue
+--     <*> optional pKeyRegistDeposit
+--     <*> optional pPoolDeposit
+--     <*> optional pMinPoolCost
+--     <*> optional pEpochBoundRetirement
+--     <*> optional pNumberOfPools
+--     <*> optional pPoolInfluence
+--     <*> optional pMonetaryExpansion
+--     <*> optional pTreasuryExpansion
+--     <*> pure Nothing
+-- =======
+    <$> optional (pProtocolVersion $ Just "new")
```

Comment:
Is this disambiguation necessary? What do these flags conflict with?

Reply:
Ah I see. I think we can inform the user in the help text as to why the `governance-action-deposit` is disambiguated and avoid doing the same for the other protocol parameter options.